src/figuresClasses.ts

```diff
@@ -1,19 +1,78 @@
-export interface Figure {
+type Shape = 'triangle' | 'circle' | 'rectangle';
+type Color = 'red' | 'green' | 'blue';
 
+function round(num: number): number {
+  return Math.floor(num * 100) / 100;
 }
 
-export class Triangle {
+function error(...arg: number[]): void {
+  if (arg.some((el) => el <= 0)) {
+    throw new Error('Error');
```

Comment:
Checklist item 3: If you throw an error try to provide its detailed description in error message.